---
fault_name: offset
min_offset: 1
max_offset: 5
default_offset: 5